core-java-modules/core-java-21/src/main/java/com/baeldung/java21/PatternCaseLabels.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.java21;
+
+public class PatternCaseLabels {
+
+    static String processInputOld(String input) {
+        String output = null;
```

Comment:
There's no need to initialize the variable. It's redundant.

Reply:
Removed the variable initialization to null.